odoo/orm/fields_relational.py

```diff
@@ -568,11 +576,11 @@ def convert_to_display_name(self, value, record):
 
     def get_depends(self, model):
         depends, depends_context = super().get_depends(model)
-        if not self.compute and isinstance(self.domain, list):
+        if not self.compute and isinstance(domain := self.domain, (list, Domain)):
```

Comment:
`domain := self.domain` is not necessary and quite confusing as walrus operator use.

Reply:
I find it simpler to read, it means that inside the block we have an alias for a typed value.